src/python/WMCore/Services/DBS/DBS3Reader.py

```diff
@@ -763,7 +767,10 @@ def listDatasetLocation(self, datasetName, dbsOnly = False):
                 return list()
 
             for blockInfo in blocksInfo:
-                locations.update([blockInfo['origin_site_name']])
+                if pnn_regex.match(blockInfo['origin_site_name']):
```

Comment:
@ticoann you have factored the pnn_regex out now into SiteDB.py (below) so this will now fail.
https://github.com/dmwm/WMCore/pull/6358/files#diff-bf6f1117d1eafd52a7c14487c2b89307R15